lib/CreepPlayers/SpawnRunPlayer.js

```diff
@@ -0,0 +1,19 @@
+
+var BaseCreepPlayer = require('./BaseCreepPlayer');
+var Spawn = require('./Behaviors/Spawn');
+var Run = require('./Behaviors/RunDumb');
+
+
+function NecromancerPlayer(creepController) {
+    this.cc = creepController;
+
+    var spawn = new Spawn(this.cc);
+    var run = new Run(this.cc);
+
+    this.behaviors = [spawn, run];
```

Comment:
in the necro player I made him ranged attack while he saw a skeleton ( or move into position to range attack ). is there a behavior that will do that?